theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeThumb.tsx

```diff
@@ -121,17 +126,7 @@ const FocusRangeThumb: React.FC<{
           focusRangeStripTheme.rangeStripMinWidth,
         )
 
-        if (handlerRef.current) {
-          originalBackground = handlerRef.current.style.background
```

Comment:
This stuff is written in the imperative style and is hard to grok/debug. Plus, if the `onStart|End|Drag` callbacks don't fire for any reason, we'll have invalid styles applied to the element.